labelbox/schema/ontology.py

```diff
@@ -241,6 +241,7 @@ class Type(Enum):
         BBOX = "rectangle"
         LINE = "line"
         NER = "named-entity"
+        RELATIONSHIP = "edge"
```

Comment:
Could you update the notebooks with `Tool.Type.RELATIONSHIP`? For instance, I see `ontologies.ipynb` lists out all possible Tool types

Reply:
updated 4 notebooks:
- examples/annotation_import/image.ipynb
- examples/annotation_import/pdf.ipynb
- examples/basics/ontologies.ipynb
- examples/prediction_upload/video_predictions.ipynb